src/neo/SmartContract/ApplicationEngine.Crypto.cs

```diff
@@ -83,5 +117,43 @@ protected internal bool CheckMultisig(byte[][] pubkeys, byte[][] signatures)
             }
             return true;
         }
+
+        /// <summary>
+        /// The implementation of System.Crypto.PointAdd.
+        /// Add operation of two gt points.
+        /// </summary>
+        /// <param name="gt1">Gt1 point as byteArray</param>
+        /// <param name="gt2">Gt1 point as byteArray</param>
+        /// <returns></returns>
+        protected internal byte[] Bls12381Add(byte[] gt1, byte[] gt2)
```

Comment:
Maybe these methods should be in native contract `CryptoLib`?

Reply:
why?